core/root/cluster.go

```diff
@@ -468,23 +466,34 @@ func (c clusterImpl) tags() map[string]string {
 	return c.controlPlane.Cluster.StackTags
 }
 
-func (c clusterImpl) Update(targets OperationTargets) (string, error) {
+func (c clusterImpl) Apply(targets OperationTargets) error {
 	cfSvc := cloudformation.New(c.session)
 
 	exists, err := cfnstack.StackExists(cfSvc, c.controlPlane.ClusterName)
 	if err != nil {
-		logger.Errorf("please check your AWS Credentials/Permissions")
-		return "", fmt.Errorf("can't lookup AWS CloudFormation stacks")
+		logger.Errorf("please check your AWS credentials/permissions")
+		return fmt.Errorf("can't lookup AWS CloudFormation stacks: %s", err)
 	}
-	if !exists {
-		logger.Errorf("you can only 'update' clusters with a matching cloudformation stack")
-		return "", fmt.Errorf("missing cluster root stack %s", c.controlPlane.ClusterName)
+
+	if exists {
+		report, err := c.update(cfSvc, targets)
+		if err != nil {
+			return fmt.Errorf("error updating cluster: %v", err)
+		}
+		if report != "" {
+			logger.Infof("Update stack: %s\n", report)
```

Comment:
How about `stack update started: %s`, to make it even more clear that the update isn't completed yet

Reply:
Hasn't update already run at this point and produced a report?

Reply:
Oops, you are correct! Then it should be `stack updated` or maybe `stack update completed` or alike. Not yet sure what option would be most consistent with all the other log messages.